eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/http/push/AsyncHTTPPushRequest.java

```diff
@@ -175,7 +172,7 @@ public void tryHTTPRequest() {
         body.add(new BasicNameValuePair(PushMessageRequestBody.EXTFIELDS,
             JsonUtils.toJSONString(EventMeshUtil.getEventProp(handleMsgContext.getEvent()))));
 
-        HttpEntity httpEntity = new UrlEncodedFormEntity(body, StandardCharsets.UTF_8);
```

Comment:
Would it be better to use the one that comes with jdk

Reply:
Thanks for your review.
I'm not sure that is better. `StandardCharsets.UTF_8` is jdk provided natively, but it can't indicate the default meaning in EventMesh comparing with `Constants.DEFAULT_CHARSET`. I guess this may be the motivation for EventMesh to define this field.

Reply:
This one is eventmesh defined and is the same.
![image](https://user-images.githubusercontent.com/91315508/234152816-23022b1c-0df4-4606-ae86-ad9850829226.png)